gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineJavaVersionIntegrationTest.groovy

```diff
@@ -45,33 +48,6 @@ class BaselineJavaVersionIntegrationTest extends IntegrationSpec {
 
     File mainJava
 
-    // language=Gradle
```

Comment:
contents of this are the same below.  just cleaner to do it all in setup method since nothing uses the standardBuildFile variable.